src/protocol/data/memcache/response.c

```diff
@@ -52,6 +54,13 @@ response_create(void)
         return NULL;
     }
 
+    rsp->vbuf = cc_alloc(rsp_val_buf_size);
```

Comment:
RSP_VAL_BUF_SIZE is relatively large. If we allocate one such buffer per response (which I've been assigning/caching one per connection), you may end up using a lot of memory when the connection count is high. E.g. my current "high connection count" test has 10K+ connections so that would mean 10GB buffer being allocated.

In the single-threaded context, and size-adjustable `wbuf` for each stream (currently implemented using doubling buffer), I think it's possible to only allocate this buffer once, i.e. per-thread, and copying the content into the connection's `wbuf` before we turn to the next request or connection.

To merge this PR into master, I think it would be preferably that we don't affect the resource footprint of the other binaries dependent on `memcache response` objects, although, since `pelikan_twemcache` etc don't ever access this field, I don't think they will affect the RSS, so at least the current test jobs won't be OOM-killed.